modules/claims_api/spec/workers/claim_establisher_spec.rb

```diff
@@ -74,4 +74,30 @@
       subject.new.perform(claim.id)
     end.to raise_error(ActiveRecord::RecordInvalid)
   end
+
+  it 'preserves original data upon error' do
+    orig_data = claim.form_data
+    body = [{ 'key' => 400, 'severity' => 'FATAL', 'text' => nil }]
+    allow_any_instance_of(EVSS::DisabilityCompensationForm::Service).to(
+      receive(:submit_form526).and_raise(Common::Exceptions::BackendServiceException.new)
+    )
+    subject.new.perform(claim.id)
+    claim.reload
+    expect(claim.evss_id).to be_nil
+    expect(claim.evss_response).to eq(body)
+    expect(claim.status).to eq(ClaimsApi::AutoEstablishedClaim::ERRORED)
+    expect(claim.form_data).to eq(orig_data)
+  end
+
```

Comment:
Need to also add a test for when the `::EVSS::DisabilityCompensationForm::ServiceException` is raised.